spec/inertia/response_spec.rb

```diff
@@ -1,10 +1,42 @@
-RSpec.describe 'Inertia::Response', type: :request do
+RSpec.describe 'InertiaRails::Response', type: :request do
   describe 'inertia location response' do
     it 'returns an inertia location response' do
       get my_location_path
 
       expect(response.status).to eq 409
       expect(response.headers['X-Inertia-Location']).to eq empty_test_path
     end
-  end  
+  end
+
+  describe 'inertia_redirect_to' do
+    context 'without an :errors option' do
+      it 'behaves like a regular redirect_to' do
+        post regular_inertia_redirect_to_path
+        expect(response.status).to eq 302
+        expect(response.headers['Location']).to eq(empty_test_url)
+        expect(session[:inertia_errors]).not_to be
+      end
+    end
+
+    context 'with an :errors option' do
+      # In practice, a GET -> redirect + errors probably shouldn't happen
```

Comment:
I was on the fence on whether to forward the session errors on a get request. It would be an atypical pattern in the controller, but I figured it was better to make the behavior consistent for different request types